events/message.js

```diff
@@ -3,12 +3,14 @@ const cotwActions = require('../eventActions/cotwActions');
 const hocActions = require('../eventActions/hocActions');
 const snapshotActions = require('../eventActions/snapshotActions');
 const sleepclubActions = require('../eventActions/sleepclubActions');
-const profanityActions = require('../eventActions/profanityActions');
+////////////////////////////////////////////////const profanityActions = require('../eventActions/profanityActions');
```

Comment:
Did you ///////// comment this out /////////// enough?

Reply:
Next time I need more `/////////////////////////////////////////`